### PR TITLE
0.27.1 doesn't build

### PR DESCRIPTION
**Pull request by klaussfreire**

### CODE REVIEW

**Review by tzulberti-jampp — APPROVED** [review]

**Review by tzulberti-jampp — COMMENTED** [review]
Shouldn't we do a hotfix release with this?

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 lz4
 xxhash
 numpy
-Cython>=0.22
+Cython>=0.22,!=0.27.1
```
Comment (tzulberti-jampp) [inline requirements.txt]: I updated to 0.27.3 and still got the following compilation error:

``` 
Error compiling Cython file:
------------------------------------------------------------
...
                                    self.index[startpos,1],
                                    pybuf.len) == bkey:
                                return True
                            startpos += 1
                finally:
                    PyBuffer_Release(cython.address(pybuf))
                                          ^
------------------------------------------------------------

sharedbuffers/mapped_struct.py:4055:43: Taking address of non-lvalue (type Python object)
```

Comment (klaussfreire) [reply requirements.txt]: Seems like a different error. Will look into it on monday

Comment (tzulberti-jampp) [reply requirements.txt]: Ok, I will create a new ticket for this error: https://github.com/jampp/sharedbuffers/issues/23

Comment (klaussfreire) [reply requirements.txt]: It seems like another bug in Cython

**Review by tzulberti-jampp — APPROVED** [review]

**Review by pablodo — APPROVED** [review]

**Review by zeitos — APPROVED** [review]

**Pull request merged**